src/graph/CreateEdgeIndexExecutor.cpp

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "graph/CreateEdgeIndexExecutor.h"
+
+namespace nebula {
+namespace graph {
+
+CreateEdgeIndexExecutor::CreateEdgeIndexExecutor(Sentence *sentence,
+                                                 ExecutionContext *ectx) : Executor(ectx) {
+    sentence_ = static_cast<CreateEdgeIndexSentence*>(sentence);
```

Comment:
`dynamic_cast` is better？

Reply:
when using `dynamic_cast` transfer derived to base class, it's more safety than `static_cast`. Because the sentence is extend from `Sentence` when it declared so I think it's ok at here.